### PR TITLE
Add types definitions for chessboardjs

### PR DESCRIPTION
**Pull request by davidmpaz**
Address oakmac/chessboardjs#152

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/chessboardjs/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (plantain-00) [inline types/chessboardjs/tsconfig.json]: `strictNullChecks` should be enabled

File: types/chessboardjs/tslint.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (plantain-00) [inline types/chessboardjs/tslint.json]: This file should only `"extends": "dtslint/dt.json"`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/chessboardjs/tslint.json (hunk 2 above)
Comment (davidmpaz) [reply types/chessboardjs/tslint.json]: Hi @plantain-00, thanks for the review.

**Review by plantain-00 — APPROVED** [review]

**Review by plantain-00 — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**